packages/js/src/ui/components/elements/Header/MoreActionsOptions.tsx

```diff
@@ -12,7 +12,7 @@ export const MoreActionsOptions = () => {
       <ActionsItem
         label="Mark all as read"
         /**
-         * TODO: Implement setFeedOptions and isSelected after Filter is implemented
+         * TODO: Implement setNotificationsOptions and isSelected after Filter is implemented
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** We can remove this comment, the onClick handler has been implemented now‏